api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -279,12 +291,17 @@ message JwtRequirement {
     // All of them must pass, if one of them fails or missing, it fails.
     JwtRequirementAndList requires_all = 4;
 
+    // Set both allow_missing and allow_invalid instead to have the same effect.
```

Comment:
Should this field now be tagged as `deprecated`?

Reply:
I reverted this. I think it's better to keep the current `allow_missing_or_failed` and add `allow_missing` (but not failed). I don't think the mode `allow_failed` make any sense by itself, so keeping `allow_missing_or_failed` is better than deprecate it and replace with `allow_missing` and `allow_failed` as suggested previously in the related [issue](https://github.com/envoyproxy/envoy/issues/8909).